src/ansys/pre_commit_hooks/add_license_headers.py

```diff
@@ -105,6 +108,78 @@ def set_lint_args(parser: argparse.ArgumentParser) -> argparse.Namespace:
     return parser.parse_args()
 
 
+def get_full_paths(file_list: list) -> list:
+    """
+    Update file paths to be absolute paths with system separators.
+
+    Parameters
+    ----------
+    file_list: list
+        List containing committed files.
+
+    Returns
+    -------
+    list
+        List containing the full paths of committed files.
+    """
+    full_path_files = []
+    for file in file_list:
+        if "win" in sys.platform:
+            split_str = file.split("/")
+            full_path_files.append(os.path.abspath(os.path.join(*split_str)))
+        else:
+            full_path_files.append(os.path.abspath(file))
+
+    return full_path_files
+
+
+def update_license_file(arg_dict: dict) -> int:
+    """
+    Update the LICENSE file to match MIT.txt, adjusting the year span to each repository.
+
+    Parameters
+    ----------
+    arg_dict: dict
+        Dictionary containing the committed files, custom copyright, template, license,
+        changed_headers, start & end year, and git_repo
+    """
+    # Get location of LICENSE file in the repository the hook runs on
+    git_root = arg_dict["git_repo"].git.rev_parse("--show-toplevel")
+    repo_license_loc = os.path.join(git_root, "LICENSE").replace(os.sep, "/")
+    save_repo_license = shutil.copyfile(repo_license_loc, f"{repo_license_loc}_save")
+
+    # Get the location of MIT.txt in the hook's assets folder
+    hook_loc = pathlib.Path(__file__).parent.resolve()
+    hook_license_file = os.path.join(hook_loc, "assets", "LICENSES", f"{DEFAULT_LICENSE}.txt")
```

Comment:
```suggestion
    hook_license_file = hook_loc / "assets" / "LICENSES" / f"{DEFAULT_LICENSE}.txt"
```
Path objects can be chained like this... avoid using ``os.path`` overall

Reply:
This applies everywhere